src/creatures/players/player.h

```diff
@@ -1789,6 +1789,18 @@ class Player final : public Creature, public Cylinder
 			}
 			return false;
 		}
+		void setImmuneFear(ConditionType_t conditiontype) {
+			fearCondition.first = conditiontype;
+			fearCondition.second = OTSYS_TIME() + 10000;
+		}
+		bool isImmuneFear(ConditionType_t conditiontype) {
+			uint64_t timenow = OTSYS_TIME();
+			if ((fearCondition.first == conditiontype)
+                    && (timenow <= fearCondition.second)) {
+				return true;
+			}
+			return false;
+		}
```

Comment:
As the `contitiontype` here will always be `CONDITION_FEARED`, we can omit that parameter
```suggestion
		void setImmuneFear() {
			fearCondition.first = CONDITION_FEARED;
			fearCondition.second = OTSYS_TIME() + 10000;
		}
		bool isImmuneFear() {
			uint64_t timenow = OTSYS_TIME();
			if ((fearCondition.first == CONDITION_FEARED)
                    && (timenow <= fearCondition.second)) {
				return true;
			}
			return false;
		}
```